lib/internal/fs/streams.js

```diff
@@ -118,6 +178,8 @@ function ReadStream(path, options) {
   this.flags = options.flags === undefined ? 'r' : options.flags;
   this.mode = options.mode === undefined ? 0o666 : options.mode;
 
+  importFd(this, options);
```

Comment:
I think that would not help a future debugging to have some properties defined in the different fashion than the others. What doo you think about creating a new internal class `FileHandleCompatibleStream` that both `ReadStream` and `WriteStream` would extend?

Reply:
I took extra care to be as compatible as possible, no internal properties have different meaning, ie `stream.fd` is still the numerical file descriptor, `stream[kFs]` is still the file operations object, there is only one new property, `stream[kHandle]` with the `FileHandle` object

Reply:
```suggestion
  ReflectApply(FIleHandleCompatibleStream, this, [options]);
```